docs/README.md

```diff
@@ -81,6 +82,7 @@ To use widget without `shop-review-badge` to landing pages, for example, you can
 |  `height`  | `string` | Change height of template |  `undefined`   |
 |  `theme`  | `string` | Change theme of template `light` or `dark` |  `undefined`   |
 |  `stars`  | `string` | Change stars stock of template |  `undefined`   |
+|  `tag`  | `string` | Filter reviews based on tag |  `undefined`   |
```

Comment:
I think this description is kind of misleading because we're only adding the tag. The proper filtering is done elsewhere, right? Is it a feature of Trustpilot? Do you have any code samples or documents explaining how the filter works?

Reply:
@danzanzini  pls find the related documentation in below links.

https://support.trustpilot.com/hc/en-us/articles/203840856-Tag-your-service-reviews
https://support.trustpilot.com/hc/en-us/articles/204868067